src/lib/seam/components/AccessCodeDetails/AccessCodeDetails.tsx

```diff
@@ -12,12 +12,14 @@ import { useIsDateInPast } from 'lib/ui/use-is-date-in-past.js'
 export interface AccessCodeDetailsProps {
   accessCodeId: string
   onBack?: () => void
+  disableLockUnlock?: boolean
   className?: string
 }
 
 export function AccessCodeDetails({
   accessCodeId,
   onBack,
+  disableLockUnlock = false,
```

Comment:
Not sure if these tend to be used individually—if not, we can remove this default value

Reply:
Default is nice at the top level API for props where undefined is not a meaningful value. This means internal components will always be passed a definite value and since it's a required prop on the internal components the type checker will ensure we don't forget to pass drill it down

Reply:
I'm keeping the default under the `DeviceDetails` component for now but let me know if it should be added elsewhere.

Reply:
These are top level because they are exported and documented as the public component api https://github.com/seamapi/react/blob/main/src/lib/seam/components/index.ts